src/main/java/com/stuypulse/robot/Constants.java

```diff
@@ -55,4 +81,81 @@ public interface Constraints {
 		double MAX_ANGULAR_VEL = (MAX_VEL * 2) / TRACK_WIDTH_METERS;
 		double MAX_ANGULAR_ACC = (Math.pow(MAX_ACC, 2)) / (TRACK_WIDTH_METERS/2); // a = v^2 / r
 	}
+
+	public interface EdwinSettings {
+        // If speed is below this, use quick turn
+        SmartNumber BASE_TURNING_SPEED = new SmartNumber("Driver Settings/Base Turn Speed", 0.4);
+
+        // Low Pass Filter and deadband for Driver Controls
+        SmartNumber SPEED_DEADBAND = new SmartNumber("Driver Settings/Speed Deadband", 0.05);
+        SmartNumber ANGLE_DEADBAND = new SmartNumber("Driver Settings/Turn Deadband", 0.05);
+
+        SmartNumber SPEED_POWER = new SmartNumber("Driver Settings/Speed Power", 1.0);
+        SmartNumber ANGLE_POWER = new SmartNumber("Driver Settings/Turn Power", 1.0);
+
+        SmartNumber SPEED_FILTER = new SmartNumber("Driver Settings/Speed Filtering", 0.2);
+        SmartNumber ANGLE_FILTER = new SmartNumber("Driver Settings/Turn Filtering", 0.01);
+
+        // Current Limit for the motors
+        int CURRENT_LIMIT_AMPS = 60;
+
+        // If the motors are inverted
+        boolean IS_INVERTED = true;
+
+        // Width of the robot
+        double TRACK_WIDTH = Units.inchesToMeters(30.0); // SEAN PROMISED !
+
+        boolean USING_GRAYHILLS = false;
+        boolean USING_GYRO = true;
+
+        public interface Motion {
+
+            double MAX_VELOCITY = 1.0;
+            double MAX_ACCELERATION = 1.0;
+
+            public interface Feedforward {
+                double kS = 0.367; // TODO: characterize
+                double kV = 2.07; // TODO: characterize
+                double kA = 0.47; // TODO: characterize
+            }
+
+            public interface PID {
+                double kP = 0.0198; // TODO: characterize
+                double kI = 0; // TODO: characterize
+                double kD = 0; // TODO: characterize
+            }
+        }
+
+        public interface Odometry {
+            Translation2d STARTING_TRANSLATION = new Translation2d();
+            Rotation2d STARTING_ANGLE = new Rotation2d();
+
+            Pose2d STARTING_POSITION = new Pose2d(STARTING_TRANSLATION, STARTING_ANGLE);
+        }
+
+        // Encoder Constants
+        public interface Encoders {
+
```

Comment:
simplify this.. i think u have to make a decision as to what gear the autons run in ( i just realized now u need a compressor subssytem to always be on for gear shifting)

Reply:
so basically remove all the grayhill stuff.

you only need <>_GEAR_NEO_TOWHEEL and <>_GEAR_DISTANCE_PER_ROTATION. leave both and pick which one to set in the subsystem .. don't need switching